src/interfaces/curses/ec_curses_mitm.c

```diff
@@ -60,13 +60,17 @@ struct wdg_menu menu_mitm[] = { {"Mitm",                'M', "", NULL},
 static void curses_arp_poisoning(void)
 {
    char *method = "arp:";
-   size_t len = strlen(method);
+   size_t len1 = strlen(method);
+
+   char *default_param = "remote";
+   size_t len2 = strlen(default_param);
    
    DEBUG_MSG("curses_arp_poisoning");
 
-   strncpy(params, method, len);
+   strncpy(params, method, len1);
+   strncpy(params+len1, default_param, len2);
 
-   curses_input("Parameters :", params + len, PARAMS_LEN - len - 1, curses_start_mitm);
+   curses_input("Parameters :", params + len1, PARAMS_LEN - len1 - len2 - 1, curses_start_mitm);
```

Comment:
The problem is basically this part. I need `len1` and `len2` to avoid showing `arp:` in the input field for the parameters and correctly limit the length. I'm not that familiar with C and didn't really have a better idea. One possibility would be to use `params + 4*sizeof(char)` instead of `params + len` which would work. Would that solution be ok? The drawback is a hard-coded constant (`4` for the characters "arp:") but in this case it probably won't be a problem as it should become obvious during testing (and "arp:" will probably never change anyway).